src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -52,14 +60,25 @@ const defaultProps = {
     onSelectRow: () => {},
     isFocused: false,
     style: null,
+    comment: '',
 };
 
 function OptionRowLHN(props) {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
-    if (!optionItem) {
+    const runUseEffectOnlyOnce = useCallback(() => {
+        if (!optionItem.hasDraftComment && props.comment && props.comment.length > 0 && !props.isFocused) {
+            Report.setReportWithDraft(props.reportID, true);
+        }
         return null;
-    }
+    }, [optionItem, props.comment, props.isFocused, props.reportID]);
+
+    useEffect(() => {
+        if (!optionItem) {
+            return null;
+        }
```

Comment:
@mkarkachov As I said before, this will not prevent the crash if `optionItem` is null. That's why we have the early return on `optionItem`.

```js
if (!optionItem) {
    return null;
}
```

`SidebarUtils.getOptionData` will probably return nothing if the report is not found.